libibverbs/verbs.h

```diff
@@ -235,6 +235,8 @@ enum ibv_odp_transport_cap_bits {
 	IBV_ODP_SUPPORT_READ     = 1 << 3,
 	IBV_ODP_SUPPORT_ATOMIC   = 1 << 4,
 	IBV_ODP_SUPPORT_SRQ_RECV = 1 << 5,
+	IBV_ODP_SUPPORT_FLUSH    = 1 << 6,
```

Comment:
Why aren't these bits coming from UAPI?

Reply:
Okay. I will send a kernel patch to add `IB_UVERBS_ODP_*` flags in `include/uapi/rdma/ib_user_verbs.h` and update this PR accordingly.

Reply:
@rleon 
I've submitted the kernel patch and updated this PR.

Thanks

Reply:
I revised the kernel-headers commit after all the kernel-side patches were merged.